extensions/wikia/AdEngine/js/video/player/ui/replayOverlay.js

```diff
@@ -0,0 +1,50 @@
+/*global define*/
+define('ext.wikia.adEngine.video.player.ui.replayOverlay', [
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (doc, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.video.player.ui.replayOverlay',
+		replayOverlayClass = 'replay-overlay';
+
+	function add(video) {
+		var overlay = doc.createElement('div');
+
+		overlay.classList.add(replayOverlayClass);
+		overlay.addEventListener('click', function () {
+			video.play();
+			log(['replay', log.levels.debug, logGroup]);
+			// make overlay invisible
+			overlay.style.width = '';
+		});
+
+		video.addEventListener('wikiaAdCompleted', function() {
```

Comment:
we are un-hiding overlay by assigning it a width when the video ends. When it starts, we want to hide it again